modules/ROOT/pages/policy-template-source.adoc

```diff
@@ -4,27 +4,24 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: policy, CloudFront
 
-[NOTE]
---
-The following section describes configurations parameters that are relevant for only Mule versions 4.1.7 and later, upto 4.2.0 and 4.2.2 and later, upto 4.3.0.
---
+The configuration parameters described in this section are relevant to only Mule runtime engine (Mule) versions 4.1.7 and later.
 
-When an application with a tracked flow is deployed to a Mule Runtime (Mule), the API Gateway of that Mule environment communicates with API Manager to request for the applied policies.
+When an application with a tracked flow is deployed to a Mule environment, the API gateway of that Mule environment communicates with API Manager to request for the applied policies.
 
-If policies are to be applied and the policy templates are not already present in Mule, API Gateway requests API Manager for the URLs from where to download the policy template.
+If the templates for the policies that are to be applied are not already present in Mule, API gateway requests API Manager for the URLs from where to download the template.
 
-You can configure the following property to provide Policy Templates to API Manager only from a specified source:
+You can configure the following property to provide Policy templates to API Manager from a specified source:
 
 `anypoint.platform.policy_template_source`
 
 [%header,cols="2*a"]
 |===
 |Value | Description
-|external | Indicates that the policy templates are to be retrieved from the legacy system, This is the default value to maintain compatibility with previously configured firewalls.
-|internal | Indicates that the policy templates are to be retrieved from the new system. This option provides additional reliability due to internal rerouting of the requests, when a server becomes unavailabile.
+|external | Policy templates to be retrieved from the legacy system. This is the default value to maintain compatibility with previously configured firewalls.
+|internal | Policy templates to be retrieved from the new system. This option provides additional reliability due to internal rerouting of the requests when a server becomes unavailable.
 |===
 
-If we start a v4.2.2 Mule Runtime, as shown in the box below, API Manager provides URLs for downloading the policy templates from Mule's CloudFront service.
+If you start Mule version 4.2.2 , as shown in the example, API Manager provides the URLs for downloading the policy templates from the Mule CloudFront service.
```

Comment:
Does the example really show how the syntax configuration defined above works in practice?

Reply:
Suggested rewording:
```suggestion
The following example shows a Mule 4.2.2 environment in which API Manager provides the URLs for downloading policy templates from the Mule CloudFront service:
```

Reply:
Why is Mule CloudFront service not mentioned in the topic intro? Is it always the template source?